third-culture-software/bhima
# Enable merging lots

This updates lets the user merge lots from the Lots Registry.  

This is a preliminary release to get some feedback on coding, UIs, etc.  I'm continuing to work on the release.  Once those issues have been addressed, I've added some integration tests, and it is ready to go, I'll remove the [Do No Merge] flag.

### Testing
- Use the IMCK database
- Go to the Lots Registry
- Update the search to find lots named:  arp
- Click on the action menu and select the last item (Find Duplicate Lots).
   This finds all lots that have matching labels and inventory_uuid-s. 
- Note that there is a column for the lot to keep (initially set to the current lot) and
   a column for 'Merge'.  
- The selected lot is marked in green.
- Select any lot you wish to merge with the selected lot.
- Click on the [Merge] button
- When you get back to the lots registry, try the merge operation on the same lot.  Notice that no matching will be found.
- Clear the search and try the merge operation on a random  lot.  Very likely not matching lots will be found.
